mediapipe/calculators/geti/inference/BUILD

```
@@ -41,21 +41,27 @@ cc_library(
         "classification_calculator.cc",
         "detection_calculator.cc",
         "instance_segmentation_calculator.cc",
+        "keypoint_detection_calculator.cc",
         "openvino_inference_adapter_calculator.cc",
         "rotated_detection_calculator.cc",
         "segmentation_calculator.cc",
         "model_infer_request_image_calculator.cc",
+        "model_infer_http_request_calculator.cc",
     ],
     hdrs = [
         "anomaly_calculator.h",
         "classification_calculator.h",
         "detection_calculator.h",
         "instance_segmentation_calculator.h",
         "kserve.h",
+        "keypoint_detection_calculator.h",
         "openvino_inference_adapter_calculator.h",
         "rotated_detection_calculator.h",
         "segmentation_calculator.h",
         "model_infer_request_image_calculator.h",
+        "model_infer_http_request_calculator.h",
+        "http_payload.h",
+        "client_connection.h"
```

```suggestion
        "model_infer_http_request_calculator.h",
        "@ovms//src:httppayload"
```
Since change to OVMS was merged this should suffice now - it won't bring TFS unnecessarily